src/styles/blocks/catalog.scss

```
@@ -11,12 +14,17 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
-    width: 200px;
-    height: 300px;
+    cursor: pointer;
+    @include card-size;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $time-hover;
+
```

It's bad pactice to set transition to all. Just specify whta you need
```suggestion
    transition: transform $time-hover, color $time-hover;
```